src/reducers/general.js

```diff
@@ -40,12 +40,16 @@ const general = (state = {
       return Object.assign({}, state, stateUpdate);
     case types.UPDATE_PATHNAME:
       return Object.assign({}, state, {
-        pathname: action.pathname
+        pathname: action.data
```

Comment:
Is this necessary to make the language feature work? All the `UDPATE_PATHNAME` actions i can find seem to use action.pathname, so I'm wondering where action.data will come from for this action.

Reply:
@charlie-jones this was marked as resolved but not changed back to `action.pathname`; is `action.data` here part of how you intend this feature to work? If so, I still wonder about the above question. Thanks!

Reply:
Very sorry about that, just changed it!